src/main/java/testapp/core/Student.java

```diff
@@ -0,0 +1,70 @@
+package testapp.core;
+
+import com.google.appengine.api.datastore.Entity;
+
+import java.util.List;
+
+/**
+ * @author Vasil Mitov <v.mitov.clouway@gmail.com>
+ */
+public class Student {
```

Comment:
I would recomend using the user naming for clear purpose for this object to the app